modules/showContact.js

```diff
@@ -0,0 +1,12 @@
+export default function showContact() {
```

Comment:
- [ ] [OPTIONAL] I kindly suggest that you make use of arrow functions because it a project requirement.

## EXAMPLE
export default () => console.log("Hello World.");

// If you want named exports, you have to export it seperately
const showContact = () => console.log("This is an app.");
export default showContact;